scripts/booksHandler.js

```diff
@@ -1,36 +0,0 @@
-import { GOOGLE_URL } from "./variables.js";
-import { createCard } from "./card.js";
-
-//function to get a book object with some information to display
-export const getBookData = (book) => {
-  let checkAuthors = book.volumeInfo.authors;
-  if (checkAuthors.length > 2) {
-    checkAuthors = `${checkAuthors[0]} et al.`;
-  }
```

Comment:
This also breaks if authors isn't an array. Again maybe optional chaining or even passing the entire authors object into a function to format it as needed

Reply:
I suggest a formatAuthors function